components/server/src/auth/generic-auth-provider.ts

```diff
@@ -258,6 +259,7 @@ export class GenericAuthProvider implements AuthProvider {
         const strategyName = this.strategyName;
         const clientInfo = getRequestingClientInfo(request);
         const cxt = LogContext.from({ user: request.user });
+        const authHost = request.get('host')
```

Comment:
that's not the wanted `host`.

we need to pick it from the session, see ll.270 `authFlow` should contain it.

Reply:
btw. this is gitpod's host name.

Reply:
Oh... Good to know :sweat_smile: